### ISSUE TITLE
Maintainer wanted!

### ISSUE THREAD

**Issue by steveruizok** [steveruizok/perfect-freehand-dart#10]
Hello, I'd like to find a new maintainer for this library. I wrote this port of my JavaScript perfect-freehand algorithm as a learning exercise, however I now see that it has been picked up fairly well within the Flutter community. 

If you would like to help maintain the library, please let me know in this issue or at steve@tldraw.com.

Specifically, I'd like to find a maintainer who would be interested in improving its examples, keeping it modern with respect to the Dart language, and keeping it more or less at parity with the JavaScript version. If you'd like to make improvements to the core algorithm, then that would be good too!

Either way, thanks for checking out perfect-freehand, I hope you've found it useful!

**Comment by tootac** [comment]
Hello. I would love to.

**Comment by logickoder** [comment]
I would love to be a maintainer

**Comment by bansalgaurav852** [comment]
I would love to be a maintainer.

**Comment by adil192** [comment]
@steveruizok I've re-ported the [steveruizok/perfect-freehand](https://github.com/steveruizok/perfect-freehand) repo to dart from scratch.
You can find the repo here: [adil192/perfect_freehand](https://github.com/adil192/perfect_freehand)

It's therefore fully up to date with the js version. I'm going to do a little bit more testing but I'm planning for this to be a drop-in replacement for [steveruizok/perfect-freehand-dart](https://github.com/steveruizok/perfect-freehand-dart) (maybe even replacing the `perfect_freehand` package on pub.dev).

**Comment by steveruizok** [comment]
Wow! If this is better than what's here in the repo, I'd be happy to discuss bringing that code here and making you a maintainer.

**Comment by adil192** [comment]
Update: I've got the demo running now! https://adil192.github.io/perfect_freehand/

**Comment by adil192** [comment]
@steveruizok
I fixed the above edge case with https://github.com/adil192/perfect_freehand/commit/0bb5a99c47027bc6980837a0dcfc5f5d5a408da2.
I would say my repo is ready to be copied into this repo now if you're ready :)

**Comment by alestiago** [comment]
What's the status of this? Who is maintaining this repository? Is this still the main repository or has it been moved elsewhere?

**Comment by adil192** [comment]
> What's the status of this? Who is maintaining this repository? Is this still the main repository or has it been moved elsewhere?

Use the package on [pub.dev](https://pub.dev/packages/perfect_freehand) for now.
There aren't any major new features or fixes in the new port, but I believe it has a friendlier API that'll also cut down on wrapper code in dependent projects (like how I needed to implement a `StrokeOptions` class in [Saber](https://github.com/saber-notes/saber)).

This (`steveruizok/perfect-freehand-dart`) is still the main repo. I was waiting for steveruizok's response but I've just made the PR now.
If we can't get the updates merged into this repo, I'll publish it to pub.dev as a separate package.
